### PR TITLE
Update Chrome to milestone 84

### PR DESCRIPTION
**Pull request by gsouquet**
The chrome team has released a new version yesterday!
This PR updates the browsers data

### CODE REVIEW

File: browsers/chrome.json [hunk 1]
```diff
@@ -574,16 +574,23 @@
...
         },
         "85": {
+          "status": "beta",
+          "engine": "Blink",
+          "engine_version": "85"
+        },
+        "86": {
           "status": "nightly",
           "engine": "Blink",
           "engine_version": "85"
```
Comment (ddbeck) [inline browsers/chrome.json:596]: Unless something very unusual has happened, I'd expect this version to have changed as well:

```suggestion
          "engine_version": "86"
```

File: browsers/chrome_android.json [hunk 2]
```diff
@@ -411,16 +411,23 @@
...
         },
         "85": {
+          "status": "beta",
+          "engine": "Blink",
+          "engine_version": "85"
+        },
+        "86": {
           "status": "nightly",
           "engine": "Blink",
           "engine_version": "85"
```
Comment (ddbeck) [inline browsers/chrome_android.json:433]:
```suggestion
          "engine_version": "86"
```

File: browsers/webview_android.json [hunk 3]
```diff
@@ -402,16 +402,23 @@
...
         },
         "85": {
+          "status": "beta",
+          "engine": "Blink",
+          "engine_version": "85"
+        },
+        "86": {
           "status": "nightly",
           "engine": "Blink",
           "engine_version": "85"
```
Comment (ddbeck) [inline browsers/webview_android.json:424]:
```suggestion
          "engine_version": "86"
```

**Comment by gsouquet** [comment]
Updated! Thank you @ddbeck for your thorough review as always

**Comment by jpmedley** [comment]
Guys,

I realize I'm asking for extra work, but beta will not be out until Thursday. Even if it's only six days, this is incorrect.

**Comment by gsouquet** [comment]
> I realize I'm asking for extra work, but beta will not be out until Thursday. Even if it's only six days, this is incorrect.

No problems at all, I've updated the PR 👍 
For future reference, where can I find when a milestone goes from nightly to beta?

**Comment by jpmedley** [comment]
[Here's the schudule](https://chromiumdash.appspot.com/schedule). This is what we *plan* to do. We hit those dates about 98% of the time. All releases are posted on the [Chrome Releases blog](https://chromereleases.googleblog.com/). It does not do a very good job of distinguishing major from minor releases. So be careful about the dates you take from it. If the date of a post isn't within a day or two of the date on the schedule, you're probably looking at a minor release.

**Comment by Elchi3** [comment]
> beta will not be out until Thursday. 

Is it out now?

I guess preparing a PR, waiting until things become reality, and only merging then is fine. No need to change things back and forth.

**Comment by ddbeck** [comment]
This was superseded by #6445, so I'm going to close it. Thanks for your help here, @gsouquet!

**Pull request closed without merging**